handler/src/main/java/io/netty/handler/ssl/Ciphers.java

```diff
@@ -752,6 +758,46 @@
      */
     public static final String TLS_PSK_WITH_CAMELLIA128_SHA256 = "TLS_PSK_WITH_CAMELLIA128_SHA256";
 
+    /**
+     * TLS_RSA_WITH_ARIA_128_GCM_SHA256
+     */
+    public static final String TLS_RSA_WITH_ARIA_128_GCM_SHA256 = "TLS_RSA_WITH_ARIA_128_GCM_SHA256";
+
+    /**
+     * TLS_RSA_WITH_ARIA_256_GCM_SHA384
+     */
+    public static final String TLS_RSA_WITH_ARIA_256_GCM_SHA384 = "TLS_RSA_WITH_ARIA_256_GCM_SHA384";
+
+    /**
+     * TLS_DHE_RSA_WITH_ARIA_128_GCM_SHA256
+     */
+    public static final String TLS_DHE_RSA_WITH_ARIA_128_GCM_SHA256 = "TLS_DHE_RSA_WITH_ARIA_128_GCM_SHA256";
+
+    /**
+     * TLS_DHE_RSA_WITH_ARIA_256_GCM_SHA384
+     */
+    public static final String TLS_DHE_RSA_WITH_ARIA_256_GCM_SHA384 = "TLS_DHE_RSA_WITH_ARIA_256_GCM_SHA384";
+
+    /**
+     * TLS_ECDHE_ECDSA_WITH_ARIA_128_GCM_SHA256
+     */
+    public static final String TLS_ECDHE_ECDSA_WITH_ARIA_128_GCM_SHA256 = "TLS_ECDHE_ECDSA_WITH_ARIA_128_GCM_SHA256";
+
+    /**
+     * TLS_ECDHE_ECDSA_WITH_ARIA_256_GCM_SHA384
+     */
+    public static final String TLS_ECDHE_ECDSA_WITH_ARIA_256_GCM_SHA384 = "TLS_ECDHE_ECDSA_WITH_ARIA_256_GCM_SHA384";
+
+    /**
+     * TLS_DHE_PSK_WITH_ARIA_128_GCM_SHA256
+     */
+    public static final String TLS_DHE_PSK_WITH_ARIA_128_GCM_SHA256 = "TLS_DHE_PSK_WITH_ARIA_128_GCM_SHA256";
+
+    /**
+     * TLS_DHE_PSK_WITH_ARIA_256_GCM_SHA384
+     */
+    public static final String TLS_DHE_PSK_WITH_ARIA_256_GCM_SHA384 = "TLS_DHE_PSK_WITH_ARIA_256_GCM_SHA384";
```

Comment:
I would be "surprised" if the `ARIA` variants are supported. That said I guess it is not a hard-requirement that all these ciphers are supported. As frankly speaking ciphers may be dropped at any point and once we make these part of a public API there is not much we can do about this.